main.js

```diff
@@ -5,7 +5,8 @@ const TestCaseRouter = require('./modules/TestCasesRepository/TestCasesRouter');
 
 app.use(express.json());
 app.use('/api/test/cases', TestCaseRouter);
-
+// API suits
+app.use('/api/test/suites', TestCaseRouter);
```

Comment:
Wystawiłeś na dwóch różnych adresach jeden router -> spowoduje to wystawienie wszystkich endpointów (zawartych w routerze) podwójnie na dwóch różnych adresach. Raczej zalecam stworzenie nowego osobnego routera.